src/Main.java

```diff
@@ -5,84 +5,87 @@
     public static void main(String[] args) {
         // Declare objects
         Scanner scanner;
-        Integer num1;
-        Integer num2;
+        String num1;
+        String num2;
+        Integer int1;
+        Integer int2;
         String operation;
         Integer solution;
 
         // Read inputs
         scanner = new Scanner(System.in);
 
         System.out.print("Enter 1st number: ");
-        num1 = scanner.nextInt();
+        num1 = scanner.next();
 
-        if(num1.equals("one")) {
-            num1 = 1;
-        } else if(num1.equals("two")) {
-            num1 = 2;
-        } else if(num1.equals("three")) {
-            num1 = 3;
-        } else if(num1.equals("four")) {
-            num1 = 4;
-        } else if(num1.equals("five")) {
-            num1 = 5;
-        } else if(num1.equals("six")) {
-            num1 = 6;
-        } else if(num1.equals("seven")) {
-            num1 = 7;
-        } else if(num1.equals("eight")) {
-            num1 = 8;
-        } else if(num1.equals("nine")) {
-            num1 = 9;
-        } else if(num1.equals("ten")) {
-            num1 = 10;
+        if(num1.equalsIgnoreCase("one")) {
+            int1 = 1;
+        } else if(num1.equalsIgnoreCase("two")) {
+            int1 = 2;
+        } else if(num1.equalsIgnoreCase("three")) {
+            int1 = 3;
+        } else if(num1.equalsIgnoreCase("four")) {
+            int1 = 4;
+        } else if(num1.equalsIgnoreCase("five")) {
+            int1 = 5;
+        } else if(num1.equalsIgnoreCase("six")) {
+            int1 = 6;
+        } else if(num1.equalsIgnoreCase("seven")) {
+            int1 = 7;
+        } else if(num1.equalsIgnoreCase("eight")) {
+            int1 = 8;
+        } else if(num1.equalsIgnoreCase("nine")) {
+            int1 = 9;
+        } else if(num1.equalsIgnoreCase("ten")) {
+            int1 = 10;
         }
```

Comment:
You should add a final `else` to catch anything unexpected. For example: 
`else { System.out.printLn("Sorry, I can only read up to ten. I do not understand [" + num1 + "].");
return; }`